pom.xml

```diff
@@ -147,63 +160,35 @@
-            </dependency>
-            <dependency>
-                <groupId>org.apache.logging.log4j</groupId>
-                <artifactId>log4j-bom</artifactId>
-                <version>${version.log4j-bom}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-            <dependency>
-                <groupId>gov.nsa.datawave.contrib</groupId>
-                <artifactId>datawave-in-memory-accumulo</artifactId>
-                <version>${version.in-memory-accumulo}</version>
-                <scope>test</scope>
-                <exclusions>
-                    <!-- Exclude log4j since with spring boot we're using log4j2 -->
-                    <exclusion>
-                        <artifactId>log4j</artifactId>
-                        <groupId>log4j</groupId>
-                    </exclusion>
-                    <exclusion>
-                        <artifactId>slf4j-log4j12</artifactId>
-                        <groupId>org.slf4j</groupId>
-                    </exclusion>
-                    <!-- Exclude java servlet and JSP api jars since the hadoop versions are old and spring includes newer version -->
                     <exclusion>
                         <artifactId>servlet-api</artifactId>
                         <groupId>javax.servlet</groupId>
                     </exclusion>
-                    <exclusion>
-                        <artifactId>jsp-api</artifactId>
-                        <groupId>javax.servlet.jsp</groupId>
-                    </exclusion>
                 </exclusions>
             </dependency>
-            <!--
-                TEMPORARILY REMOVED DUE TO UPGRADE ISSUES
             <dependency>
-                <groupId>io.springfox</groupId>
-                <artifactId>springfox-swagger-ui</artifactId>
-                <version>${version.springfox}</version>
+                <groupId>org.springdoc</groupId>
+                <artifactId>springdoc-openapi-javadoc</artifactId>
+                <version>${version.springdoc}</version>
             </dependency>
             <dependency>
-                <groupId>io.springfox</groupId>
-                <artifactId>springfox-swagger2</artifactId>
-                <version>${version.springfox}</version>
+                <groupId>org.springdoc</groupId>
+                <artifactId>springdoc-openapi-security</artifactId>
+                <version>${version.springdoc}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.springdoc</groupId>
+                <artifactId>springdoc-openapi-ui</artifactId>
+                <version>${version.springdoc}</version>
             </dependency>
-            -->
         </dependencies>
     </dependencyManagement>
     <dependencies>
+        <!-- TODO: JWO: this breaks serialization of wrapped elements in MapReduceJobDescription.java and QueryLogicDescription.java -->
```

Comment:
still an issue?

Reply:
This was something I removed once.  Then someone added it back, so I commented it out and left the comment to hopefully prevent them from adding it back again...  haha.  I'll remove it.